internal/server/server.go

```diff
@@ -208,3 +210,17 @@ func translateRefData(types []sirius.RefDataItem, tmplHandle string) string {
 	}
 	return tmplHandle
 }
+
+func getPage(r *http.Request) int {
+	page := r.FormValue("page")
+	if page == "" {
+		return 1
+	}
+
+	v, err := strconv.Atoi(page)
+	if err != nil {
+		return 1
+	}
+
+	return v
+}
```

Comment:
Probably best to put this in `internal/server/search.go` so it's clearer what it's for